app/views/content/life-as-a-teacher/pay-and-benefits/bens-favourite-things-about-teaching/_article.html.erb

```diff
@@ -0,0 +1,17 @@
+<div class="row inset">
+  <section class="col col-720 col-space-l-top">
+      <p>What I particularly love about my school is the excitement and energy I can feel in the corridors. My students are enthusiastic and they push me to understand more about physics and maths too. It also helps that my colleagues are really friendly and supportive. I know that everyone is pulling in the same direction, and we have a good joke in the office.</p>
+      <p>A few years after I finished university, my old school sent out a letter to former students advertising their teacher training course. Around the same time, I went to meet one of my friends who was working at a Get Into Teaching event and got chatting about a career in teaching. It seemed like a great opportunity!</p>
+  </section>
+  <section class="col col-720">
+    <%= render Content::QuoteComponent.new(
+    text: "I like that I'm able to bring my whole self into the classroom.",
+    large: true
+  ) %>
+  </section>
+    <section class="col col-720 col-space-l-top">
```

Comment:
`<section>` and its contents need to outdent one level to line up with `</section>`